frontend/src/v5/store/drawings/drawings.types.ts

```diff
@@ -19,6 +19,8 @@
 export interface Drawing {
```

Comment:
A few inconsistencies with the base branch that annoy me.
`Drawing` instead of `IDrawing` (elsewhere it is `IContainer`, `IFederation`)
`DrawingsActionCreators` instead of `IDrawingsActionCreators` (elsewhere it is `IContainersActionCreators`, `IFederationsActionCreators`)
`DrawingActionDispatchers` instead of `DrawingsActionsDispatchers` (elsewhere it is `ContainersActionsDispatchers`, `FederationsActionsDispatchers`) Note this one has two differences in it.